app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/onboarding/email/enter_email.scala.html

```diff
@@ -33,7 +33,7 @@
    submitButton:  uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button,
    backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link,
    accountMenu: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.account_menu,
-   caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption
+   caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption_govuk
```

Comment:
I think, we need to use different approach. you can fix it using PageHeading component like 
`label = pageHeading(title, Some(messages("returns.disposal-details.caption"))),`

try this one if works okay. otherwise there is another approach too.